### DISCUSSION TITLE
Win Arrow Keys  Remap

### DISCUSSION THREAD

**Discussion by teejstroyer** [rxhanson/RectanglePro-Community#448] [Ideas]
The win arrow keys functionality offers a lot of features by default. I would like to be able to use hjkl to achieve this. If making that configurable isn't possible, it would nice to be able to add compound keymaps like hyper+t+h

**Comment by rxhanson** [comment]
Thanks for the feedback. I'll add this to my backlog for a future release. 

The compound keymaps are sort of a macOS limitation where a keyboard shortcut can only be registered with one non-modifier key. In order to implement this, Rectangle Pro would have to listen to keystrokes, and I'd actually prefer to allow hjkl (and others) be win arrow keys than listen to keystrokes.

**Reply by teejstroyer** [reply]
Awesome! Thank you
